ci/scripts/run_benchmarks.sh

```diff
@@ -75,6 +75,10 @@ do
     METADATA_BASELINE="'benchmark_version':'${BASELINE_VERSION}'"
   fi
 
+  if [[ ${FLAGS} == *"-PtestJVM=/usr/lib/jvm/java-11-openjdk-amd64/"* ]]; then
```

Comment:
I worry too many moving parts gives us more places to fail. How about we bake Java 11 into the base image similar to our test images.

See https://github.com/apache/geode-benchmarks/blob/develop/infrastructure/scripts/aws/image/packer.json